Artifacts/PgSql/Data/Security/1070-OrganizationDepartmentMetadata.sql

```diff
@@ -12,64 +12,98 @@ relationships_authorization_strategy_id INT;
+	-- Remove existing claim set overrides for OrganizationDepartment
+	DELETE FROM dbo.ClaimSetResourceClaimActionAuthorizationStrategyOverrides
+	WHERE ClaimSetResourceClaimActionId IN (
+		SELECT  csrc.ClaimSetResourceClaimActionId
+		FROM    dbo.ClaimSetResourceClaimActions csrc
+		INNER JOIN dbo.ClaimSetResourceClaimActionAuthorizationStrategyOverrides csrcaaso
+			ON csrc.ClaimSetResourceClaimActionId = csrcaaso.ClaimSetResourceClaimActionId
+		INNER JOIN dbo.ClaimSets cs
+			ON csrc.ClaimSetId = cs.ClaimSetId
+		INNER JOIN dbo.AuthorizationStrategies strat
+				ON csrcaaso.AuthorizationStrategyId = strat.AuthorizationStrategyId
+		WHERE   csrc.ResourceClaimId = organization_department_claim_id
+			AND cs.ClaimSetName IN ('SIS Vendor', 'Ed-Fi Sandbox', 'District Hosted SIS Vendor')
+			AND strat.AuthorizationStrategyName = 'NoFurtherAuthorizationRequired'
+	);
+	
+	DELETE FROM dbo.ClaimSetResourceClaimActions
+	WHERE ClaimSetResourceClaimActionId IN (
+		SELECT  csrc.ClaimSetResourceClaimActionId
+		FROM    dbo.ClaimSetResourceClaimActions csrc
+		INNER JOIN dbo.ClaimSetResourceClaimActionAuthorizationStrategyOverrides csrcaaso
+			ON csrc.ClaimSetResourceClaimActionId = csrcaaso.ClaimSetResourceClaimActionId
+		INNER JOIN dbo.ClaimSets cs
+			ON csrc.ClaimSetId = cs.ClaimSetId
+		INNER JOIN dbo.AuthorizationStrategies strat
+				ON csrcaaso.AuthorizationStrategyId = strat.AuthorizationStrategyId
+		WHERE   csrc.ResourceClaimId = organization_department_claim_id
+			AND cs.ClaimSetName IN ('SIS Vendor', 'Ed-Fi Sandbox', 'District Hosted SIS Vendor')
+			AND strat.AuthorizationStrategyName = 'NoFurtherAuthorizationRequired'
+	);
 
--- Set default authorization strategy for OrganizationDepartment to relationship-based
-SELECT AuthorizationStrategyId INTO relationships_authorization_strategy_id
-FROM dbo.AuthorizationStrategies
-WHERE AuthorizationStrategyName = 'RelationshipsWithEdOrgsAndPeople';
+	-- Set default authorization strategy for OrganizationDepartment to relationship-based
+	SELECT AuthorizationStrategyId INTO relationships_authorization_strategy_id
+	FROM dbo.AuthorizationStrategies
+	WHERE AuthorizationStrategyName = 'RelationshipsWithEdOrgsAndPeople';
 
-IF NOT EXISTS (SELECT 1 FROM dbo.ResourceClaimAuthorizationMetadatas WHERE ResourceClaim_ResourceClaimId = organization_department_claim_id)
-THEN
-    RAISE NOTICE 'Assigning default metadata for CRUD operations on OrganizationDepartment to the relationship-based authorization strategy.';
-    
-    INSERT INTO dbo.ResourceClaimAuthorizationMetadatas(ResourceClaim_ResourceClaimId, Action_ActionId, AuthorizationStrategy_AuthorizationStrategyId)
-    SELECT  organization_department_claim_id, ActionId, relationships_authorization_strategy_id
-    FROM    dbo.Actions
-    WHERE   ActionName IN ('Create', 'Read', 'Update', 'Delete');
-END IF;
+	IF NOT EXISTS (SELECT 1 FROM dbo.ResourceClaimAuthorizationMetadatas WHERE ResourceClaim_ResourceClaimId = organization_department_claim_id)
```

Comment:
ResourceClaimAuthorizationMetadatas is old table  , please update with new table . refer sql server version script